docs/nodes/credentials/Zendesk/README.md

```diff
@@ -10,6 +10,23 @@ You can find information about the operations supported by the Zendesk node on t
 
 Create a [Zendesk](https://zendesk.com/) account.
 
+## Using OAuth
+
+1. Access your Zendesk dashboard.
+2. Click on the gear icon on the left.
+3. Click on API under the ***Channels*** section.
+4. Click on the 'OAuth Clients' tab.
+5. Enter the client name in the ***Client Name*** field.
+6. Enter the description in the ***Description*** field.
+7. Copy the 'OAuth Callback URL' provided in the 'Zendesk OAuth2 API' credentials in n8n and paste it in the ***Redirect URLs*** field in the Zendesk dashboard.
```

Comment:
Wouldn't you be pasting this in the Zendesk API credentials page?

Reply:
Nice catch.